src/modules/roc_packet/target_posix/roc_packet/address.h

```diff
@@ -56,6 +59,13 @@ class Address {
     //! Check whether this is multicast address.
     bool multicast() const;
 
+    //! Return multicast interface address.
+    //!
+    //! @returns
+    //!  NULL if address is not multicast
+    //!  NULL if automatic multicast group joining/leaving is disabled
```

Comment:
> NULL if address is not multicast

We don't have this check in code.

I think it'd better to replace both lines with a single "NULL if multicast interface is not set".

Reply:
> I think it'd better to replace both lines with a single "NULL if multicast interface is not set".

Done